.github/ISSUE_TEMPLATE/BUG-REPORT.yml

```diff
@@ -0,0 +1,74 @@
+name: 🐞 Bug report
+description: File a bug/issue
+title: "[BUG] <title>"
+labels: ["type: bug", "Needs Triage"]
+body:
+- type: checkboxes
+  id: no-existing-verify
+  attributes:
+    label: I verified there's no existing issue for this bug.
+    description: Please ensure there's no existing issue for the bug before creating one.
+    options:
+    - label: There's no existing issues
+      required: true
+- type: input
+  id: skin-version
+  attributes:
+    label: Skin version
+    placeholder: Which version of Skin is the issue occurring in?
+- type: textarea
+  id: device-os-browser-info
+  attributes:
+    label: Device/OS/Browser info
+    description: Add details about your device, OS, and browser here.
+    placeholder: |
+      Add details.
+  validations:
+    required: false
+- type: textarea
+  id: current-behavior
+  attributes:
+    label: Current behavior
+    description: A concise description of what you're experiencing.
+  validations:
+    required: false
+- type: textarea
+  id: expected-behavior
+  attributes:
+    label: Expected behavior
+    description: A concise description of what you expected to happen.
+  validations:
+    required: false
+- type: textarea
+  id: steps-to-reproduce
+  attributes:
+    label: Steps to reproduce
+    description: How do you trigger this bug? Please walk us through it step by step.
+    value: |
+      1.
+      2.
+      3.
+      ...
+    render: bash
+  validations:
+    required: true
+- type: textarea
+  id: relevant-code
+  attributes:
+    label: Relevant code block
+    description: If there's a known relevant code block, paste it here.
+  validations:
+    required: false
+- type: textarea
+  id: additional-context
+  attributes:
+    label: Additional context
+    description: Links? References? Screesnhots? Anything that will give us more context about the issue you are encountering!
+
+      Tip: You can attach images or log files by clicking this area to highlight it and then dragging files in.
+  validations:
+    required: false
+- type: markdown
```

Comment:
Not sure if this last section is really necessary

Reply:
I'd like to see this in action before deciding. It's a nice touch IMO.